docs/src/UserGuide/write.md

```diff
@@ -28,6 +28,17 @@ savedataset(ds, path="ds.zarr", driver=:zarr)
 nothing # hide
 ````
 
+Save a dataset to Zarr format with compression:
+
+````@example write
+using Blosc
+n = 7 # compression level, number between 0 (no compression) and 9 (max compression)
+compression = Zarr.BloscCompressor(clevel=n)
+
+savedataset(ds, path="ds.zarr", driver=:zarr, compressor=compression)
+nothing # hide
+````
+
```

Comment:
we should have more than 1 compression level, and compare them both at the end, namely, what their disk sizes.

Reply:
You mean, compare on my computer and report sizes in the documentation?

Reply:
here, directly. Let's say a `ds_level_1.zarr` and `ds_level4.zarr`.  And then query their sizes and print them.

Reply:
I had never use this argument, so..., also I don't know what are the defaults, and hence how different things actually are.

Reply:
oh, I see I didn't even thought that the code were actually run! We'll look into it

Reply:
Humm, strangely, the compression does not seems to work as expected. There is no effect on Zarr folders and there is some on netcdf files...

Reply:
76M	test1.nc
103M test_default.nc
75M	test9.nc

84M	test1.zarr
83M	test_default.zarr
82M	test9.zarr